devlab/Vagrantfile

```diff
@@ -42,7 +42,21 @@ nodes = {
 
 Vagrant.require_version '>= 1.6.0'
 
+if not options['http_proxy'].empty?
+    unless Vagrant.has_plugin?("vagrant-proxyconf")
+        system('vagrant plugin install vagrant-proxyconf')
+    end
+end
+
 Vagrant.configure(2) do |config|
+
+  if Vagrant.has_plugin?("vagrant-proxyconf")
+    config.proxy.http     = "#{options['http_proxy']}"
+    config.proxy.https    = "#{options['https_proxy']}"
+    config.proxy.https    = "#{options['ftp_proxy']}"
+    config.proxy.no_proxy = "localhost,127.0.0.1,#{options['grizzly_ip']},#{options['icehouse_ip']},#{options['juno_ip']},#{options['nfs_ip']}"
```

Comment:
Also please keep the line length at 120 symbols, it's impossible to read such long lines on github